src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentRow.tag

```diff
@@ -43,16 +43,14 @@
     <div class="col-xs-2">
       <c:if test="${frc.editDeleteEnabled}">
         <form class="responseCommentDeleteForm pull-right">
-          <a href="<%= Const.ActionURIs.INSTRUCTOR_FEEDBACK_RESPONSE_COMMENT_DELETE %>"
+          <a <c:set var="deleteUri" value="<%= Const.ActionURIs.INSTRUCTOR_FEEDBACK_RESPONSE_COMMENT_DELETE %>" />
```

Comment:
While it works, it doesn't really look clean to set a JSP variable inside an HTML tag. Let's move it outside.